SteamProject/MySteam/CategoryHandler.cpp

```diff
@@ -63,6 +98,8 @@ FCategory FCategoryHandler::GetCategory(const int Index)
 	return FCategory();
```

Comment:
This is dangerous logic and the reason why we're using an out parameter in the Category class. You should use the same technique in this method.

Remember: **references are an alias for an existing object in memory**. In `return FCategory()` you're creating an object in local scope and returning a reference to it, but since it's in local scope, as soon as the function ends, the object's memory is released. Trying to access the returned value after that will result in undefined behavior.